source/gameshared/gs_weapondefs.cpp

```diff
@@ -11,6 +11,40 @@
 #define WEAPONUP_TIME_VERY_SLOW 1000
 
 const WeaponDef gs_weaponDefs[] = {
+	{
```

Comment:
can do `{ }` and everything will be initialised to 0

Reply:
didn't think about that

Reply:
I think this requires to replace all the accesses to the name element to have a condition for if the name is NULL (if we 0 all that out). So maybe it's not that worth it.